app_dart/lib/src/request_handlers/presubmit_luci_subscription.dart

```diff
@@ -126,4 +129,60 @@ class PresubmitLuciSubscription extends SubscriptionHandler {
     }
     return properties['presubmit_max_attempts'] as int;
   }
+
+  /// Sends [ScheduleBuildRequest] using information from a given build's
```

Comment:
All of the "new" code here is moved here inline from v1 service objects that I deleted, to make it easier to reason about the fact that the legacy code is only used in this one location. (The alternative would be to leave partial versions of the v1 service classes; I can unwind that if there's a strong preference to do it that way.)